src/textual/message_pump.py

```diff
@@ -108,6 +108,7 @@ def is_parent_active(self) -> bool:
 
     @property
     def is_running(self) -> bool:
+        """bool: Check if the message pump is running (potentially processing messages)"""
```

Comment:
As a property, possibly better documented as just "Is the..." - from the caller's perspective it's a value, not a call that does things.